code/modules/mob/living/simple_mob/subtypes/animal/hyena/hyena.dm

```diff
@@ -0,0 +1,159 @@
+/*
+
+Hyenas are any feliform carnivoran mammals of the family Hyaenidae. With only four extant species (in three genera), it is the fifth-smallest biological family in the Carnivora, and one of the smallest in the class Mammalia.
+They're also cool, and Rykka/Nyria wrote this uwu
+
+*/
+
+// Cataloguer data here
+/datum/category_item/catalogue/fauna/hyena
+	name = "Desert Fauna - Hyena"
+	desc = "Classification: Crocuta crocuta \
+	<br><br>\
+	It is primarily a hunter but may also scavenge, with the capacity to eat and digest skin, bone and other animal waste. \
+	The spotted hyena displays greater plasticity in its hunting and foraging behaviour than other similar carnivores; \
+	it hunts alone, in small parties of 2–5 individuals or in large groups. \
+	During a hunt, spotted hyenas often run through ungulate herds in order to select an individual to attack. \
+	Once selected, their prey is chased over a long distance, often several kilometres, at speeds of up to 60 km/h."
+	value = CATALOGUER_REWARD_MEDIUM
+
+// Start Defining the mob here
+/mob/living/simple_mob/animal/hyena 
+	name = "Hyena"
+	desc = "Yeen! Its fur is a dusty yellow-brown color with black spots, and it has rounded ears... and SHARP TEETH! You inexplicably want to pet it, though."
+	tt_desc = "Crocuta crocuta"
+	catalogue_data = list(/datum/category_item/catalogue/fauna/hyena)
+
+	icon = 'icons/mob/animals48x32_ch.dmi'

```

Comment:
Are they able to eat people? If so, then the correct .dmi name standard, I'd think, would be vore48x32.dmi. Add the _ch portion if you don't intend to port it upstream to Virgo. 

Otherwise, it should be added to animal_ch.dmi.